packages/node/src/methods/request-handler.ts

```diff
@@ -0,0 +1,112 @@
+import { Node } from "@counterfactual/common-types";
+import EventEmitter from "eventemitter3";
+
+import { Channels } from "../channels";
+import { NodeMessage } from "../node";
+import { IMessagingService } from "../service-interfaces";
+
+import {
+  getInstalledAppInstances,
+  getProposedAppInstances
+} from "./app-instance-operations";
+import { addAppInstance, install, proposeInstall } from "./install-operations";
+import {
+  addMultisig,
+  createMultisig,
+  getChannelAddresses
+} from "./multisig-operations";
+
+export class RequestHandler {
```

Comment:
Can we have a docstring for this? This seems pretty tightly coupled to node and I can't tell by reading this code what a request handler is

Reply:
Damn, forgot this. Good catch.